test/unit/wda-mac-specs.ts

```diff
@@ -1,40 +1,38 @@
+import { expect, use } from 'chai';
 import { WDA_MAC_SERVER } from '../../lib/wda-mac';
+import chaiAsPromised from 'chai-as-promised';
 
-describe('WDAMacServer', function () {
-  let chai;
-
-  before(async function () {
-    chai = await import('chai');
-    const chaiAsPromised = await import('chai-as-promised');
-
-    chai.should();
-    chai.use(chaiAsPromised.default);
-  });
+use(chaiAsPromised);
 
+describe('WDAMacServer', function () {
   describe('parseProxyProperties', function () {
     it('should default', function () {
-      WDA_MAC_SERVER.parseProxyProperties({}).should.eql(
+      // eslint-disable-next-line
```

Comment:
would it be possible to disable the specific lint rule instead of all of them? I assume for this one it makes sense to disable private method access lint for the whole module